src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-ui/infrastructure-hosts-page.mdx

```diff
@@ -267,7 +267,7 @@ Here are some of the things you can do from the **Network** tab:
 ## Processes tab
 
 <Callout variant="important">
-  By default, the infrastructure agent doesn't send [data about the operating system's processes](https://docs.newrelic.com/attribute-dictionary?attribute_name=&events_tids%5B%5D=8412). To enable the sending of process data set [`enable_process_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#enable-process-metrics) to `true`. To fine-tune which processes you want to monitor, configure [`include_matching_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#include-matching-metrics).
+  By default, the infrastructure agent doesn't send [data about the operating system's processes](https://docs.newrelic.com/attribute-dictionary/?event=ProcessSample) unless you use guided install. To enable the sending of process data set [`enable_process_metrics`](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/) to `true`. To fine-tune which processes you want to monitor, configure [`include_matching_metrics`](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings#include-matching-metrics).
```

Comment:
how come this isn't a relative URL?

Reply:
Good catch. Looks like they were absolute originally and I missed that in the sweep. Just pushed up a fresh commit to fix these should-be-relative links.